### DISCUSSION TITLE
Salvataggio dei log

### DISCUSSION THREAD

**Discussion by maelaborazioni** [globaleaks/globaleaks-whistleblowing-software#3724] [Q&A]
Dove vengono conservati i log? Nel database o in file di testo?
Vengono conservati per sempre o per un tempo massimo? Nel secondo caso è possibile impostare questo tempo?
Sono completi, inalterabili ed è possibile verificarne l'integrità?
Grazie

**Comment by evilaliv3** [comment]
Al momento:
1. esiste un log del webserver come file di testo sul filesystem
2. esiste un audit log applicativo memorizzato sul database al momento mantenuto indefinitamente
3. si prevede di aggiungere la possibilità di configurazione di una policy di data retention dei log del punto 2.
4. in merito a completezza, interabilità e integrità non è questo un qualcosa che tipicamente una applicazione possa garantire da sola senza un deposito presso terze parti; se può chiarirci meglio la domanda o se ha suggerimenti di implementazione ci dica pure che provvediamo ad approfondire la questione.

**Comment by maelaborazioni** [comment]
Tra le domande che ci hanno posto i legali ce ne sono alcune nella sezione **Log Management**.
Riporto dal sito _https:// www. unolegal. it/_

> Per Log Management si intende la registrazione degli accessi logici (autenticazione informatica) ai sistemi di elaborazione e archivi elettronici da parte degli amministratori di sistema. 
> 
> Le registrazioni (access log) devono avere caratteristiche di completezza, inalterabilità e possibilità di verifica della loro integrità adeguate al raggiungimento dello scopo di verifica per cui sono richieste. 
> 
> Le registrazioni devono comprendere i riferimenti temporali e la descrizione dell’evento che le ha generate e devono essere conservate per un congruo periodo, non inferiore a sei mesi.

Il terzo paragrafo risponde al 2.) della risposta che mi avete fornito ma mi mancherebbe l'informazione relativa al secondo paragrafo.

Grazie
